lib/rules/link-text.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "link-text",
```

Comment:
I think this would be more clear as `unique-link-text`. It's pretty vague as written.